packages/components/src/logos/index.tsx

```diff
@@ -23,27 +23,37 @@ export { ReactComponent as HiveCombinationMark } from './hive-combination-mark.s
 export { ReactComponent as GraphQLFoundationLogo } from './graphql-foundation.svg';
 
 export interface LettermarkLogoProps extends HTMLAttributes<HTMLElement> {}
-const createLettermarkLogo = (text: string) => {
+
+const HOCLettermarkLogo = (text: string) =>
   function LettermarkLogo(props: LettermarkLogoProps) {
     return (
       <span
         role="img"
         {...props}
         className={cn(
-          'inline-flex items-center justify-center text-sm font-medium uppercase leading-5',
+          'inline-flex items-center justify-center text-sm font-medium leading-5',
           props.className,
         )}
       >
         {text}
       </span>
     );
-  }
-  LettermarkLogo.displayName = 'LettermarkLogo';
```

Comment:
cc @hasparus I removed this because this component don't use `forwardRef` so we don't need to set `displayName`